tasks/setup-centos-7.yml

```
@@ -24,6 +24,13 @@
           - "kmod-wireguard"
           - "wireguard-tools"
         state: present
+      register: centos7_yum_updates
```

```suggestion
      register: wireguard__centos7_yum_updates
```
Internal variables should be prefixed with `wireguard__`.

As you're already on it :wink: : Could you please also add this prefix here:

https://github.com/githubixx/ansible-role-wireguard/blob/17ed7be8f324a9521193e193df3b7d2bb5c7abc5/tasks/setup-centos-7.yml#L68
https://github.com/githubixx/ansible-role-wireguard/blob/17ed7be8f324a9521193e193df3b7d2bb5c7abc5/tasks/setup-centos-7.yml#L74-L75

That was an oversight to not add it. Thanks!